4.0.0/migration.md

```diff
@@ -194,9 +331,13 @@ Stopping the MySQL to PostgreGRE migration job after 10 hours indicates that the
       deploy.go:1685] waiting for mysql to postgres migration to finish... (attempt 5/3600)
       deploy.go:1685] waiting for mysql to postgres migration to finish... (attempt 6/3600)
       deploy.go:1685] waiting for mysql to postgres migration to finish... (attempt 7/3600)
+      ...
+      deploy.go:1685] waiting for mysql to postgres migration to finish... (attempt 3600/3600)
+      deploy.go:1587] mysql-to-postgres-migration-job failed
+
 
-1. Stop the installer  with CTRL + c because the migration will be retried on the next run.
+1. It’s also safe to stop the Installer with `CTRL + c` instead of waiting for 10 hours: the migration will be retried on the next run, as it's an idempotent operation.
```

Comment:
Stop the Installer with `CTRL + C` instead of waiting for 10 hours. The migration will be retried on the next run because it is an idempotent operation.